packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.ts

```diff
@@ -355,10 +386,10 @@ export class CryptoWatchPriceFeed extends PriceFeedInterface {
         ] as [number, BN][];
       })
       .flat();
-    const startTime = endTime - this.twapLength;
+    const startTime = endTime - twapLength;
     const twapPrice = computeTWAP(priceTimes, startTime, endTime, this.web3.utils.toBN("0"));
 
-    return this.invertPrice ? this._invertPriceSafely(twapPrice) : twapPrice;
```

Comment:
There was really no advantage to inverting price in this method, Instead we should do it at the `get*Price` method level

Reply:
this seems risky...where are you now doing the inversion?

Reply:
There are currently three calls to `_computeTwap`:
- Line 102 in `getHistoricalPrice` which returns the TWAP'd historical price. This now inverts the return value of `_computeTwap`
- Line 266 in `update` which sets the price returned by `getCurrentPrice`. This value gets inverted in `getCurrentPrice` now, whereas previously it was being inverted in `_computeTwap`

I think this change makes much more sense and doesn't have a hidden inversion in `_computeTwap`

Reply:
that's true! invert where it's used rather than hiding it here.

Reply:
I see that the inversion now happens on line 103 right after you get the price 👌